src/components/Registration/BusinessLookup.vue

```diff
@@ -243,24 +212,19 @@ export default class BusinessLookup extends Vue {
   cursor: pointer;
   font-size: $px-16;
 }
-#bl-change-btn {
-  margin-right: -14px;
-}
 #bl-more-actions {
   margin-right: -14px;
   white-space: nowrap;
   #bl-undo-btn {
     min-width: unset;
-    border-right: 1px solid $gray1;
+    float: right;
```

Comment:
Check this on narrow screens. The button (not hover state) should line up with the right margin. When hovering, the blue background does extend into the margin.

Reply:
look good to me